plugin/grails-app/taglib/grails/plugin/springsecurity/SecurityTagLib.groovy

```diff
@@ -250,6 +257,24 @@ class SecurityTagLib implements GrailsConfigurationAware {
 		return webInvocationPrivilegeEvaluator.isAllowed(request.contextPath, url, method, auth)
 	}
 
+	protected boolean isFallback(def attrs) {
+		boolean fallback = false
+		def o = attrs.remove("fallback")
+		if (o instanceof Boolean) {
+			fallback = o
+		} else {
+			if (o != null) {
+				try {
+					def str = o.toString()
+					if (str) {
+						fallback = Boolean.parseBoolean(str)
+					}
+				} catch(e){}
```

Comment:
The `catch` block shouldn't be empty.  An error should be logged

Reply:
Do you think it'd be better to log the error and default `fallback` to false or to throw a tag exception indicating an invalid type was passed in as the fallback attribute?

Reply:
I'm just logging the error for now, let me know if you think we should throw the exception instead.

Reply:
Apologies.  After reviewing what's currently done in `SecurityTagLib`, I think it would be more consistent if the `try/catch` block were removed and, if there is an exception, it's allowed to propagate through the system.  Thoughts?

Reply:
Fine by me! Just pushed out that change.